Formula/o/openstackclient.rb

```diff
@@ -6,6 +6,7 @@ class Openstackclient < Formula
   url "https://files.pythonhosted.org/packages/4b/4a/4fa8b9e681dbd38ecf2a86e05f0f9fa48eb4e31e7c4f5925877367022127/python-openstackclient-6.3.0.tar.gz"
   sha256 "9ba0bdf8d03087e585b40a8dcc429cebbdd5fdec24c1d60a102bf9f331320df1"
   license "Apache-2.0"
+  revision 1
```

Comment:
Since this I the first mention I've seen of this I don't think a revision bump is needed.
```suggestion
```

Reply:
Oh, I thought it was necessary when not bumping version. I’ll remove the line.

Reply:
It is required to force an update for users, so we use it when that is needed. In this case I don't think it is though.